test/src/main/resources/DeploymentSpecifics.properties

```diff
@@ -82,7 +82,14 @@ web.protocol=http
 web.contextpath=/iaf-test
 #web.contextpath=${instance.name.lc}
 
-# nl.nn.adapterframework.pipes.CredentialCheckingPipe 
+# active.authentication controls CredentialFactory related tests:
+# The operation of the CredentialFactory via authAlias is tested with nl.nn.adapterframework.pipes.CredentialCheckingPipe
+# The operation of expanding credentials in properties like ${credential:password:testalias}
+# For these tests to be successful, a CredentialFactory needs to be configured in the applications server.
+# When no credentialFactory is configured, the WebSphere credentialFactory is used.
+# The supplied credentials in 'credentials.properties' can be used by setting the system property:
+#   credentialFactory.class=nl.nn.credentialprovider.PropertyFileCredentialFactory
+#   authAliases.expansion.allowed=testalias
 active.authentication=true
```

Comment:
Moet deze dan niet standaard uit staan? Of moet iedereen dit instellen?

Reply:
Het is voor iedereen mogelijk dit in te stellen

Reply:
Zou je dit dan ook in testing-with-test-iaf kunnen zetten?